dask/dataframe/methods.py

```diff
@@ -443,11 +443,13 @@ def assign_index(df, ind):
 
 
 def monotonic_increasing_chunk(x):
-    data = x if is_index_like(x) else x.iloc
-    return pd.DataFrame(
-        data=[[x.is_monotonic_increasing, data[0], data[-1]]],
-        columns=["monotonic", "first", "last"],
-    )
+    if len(x) == 0:
+        # if input is empty, return empty df for chunk
+        data = None
+    else:
+        data = x if is_index_like(x) else x.iloc
+        data = [[x.is_monotonic_increasing, data[0], data[-1]]]
+    return pd.DataFrame(data=data, columns=["monotonic", "first", "last"])
```

Comment:
This will default to `object` dtype for all columns, does this have any adverse affects in the aggregate function? From reading the code I'd guess this would result in all columns being promoted to object dtype during concatenation, which may be undesired.

Reply:
Oh good point! I just did a little test and it looks like the dtype of the columns after concatenation is not object. I'm not 100% sure how this works. But I did try calling `.is_monotonic` on a series with dtype=object, and it throws a warning before coercing to numeric, so we would know if that were happening.